sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ParameterAssignedTo.vb

```diff
@@ -1,8 +1,11 @@
-﻿Imports System
+﻿
```

Comment:
_minor_ additional empty line at the start of the file

Reply:
That's intentional VB convention/habit to separate all blocks, this situation included. There used to be 
`Option Strict/Explicit/Compare/Infer On/Off`
as first line of the file followed by an empty line and imports.